Variant subsystem label fix resolves #479

akeeste opened the pull request:
This PR fixes issue #479 where the 'Drag body' label appears on a rigid body if the Variant variables are not yet defined. This causes confusion and can make Simulink bodies appear to be a different type than they actually are.

This fix uses the 'Allow zero active variant controls' option in the rigid body which allows no label to be assigned until a body is one of the Variants is correctly activated. This requires the library to be in R2017b or later.

kmruehl commented:
> 
> 
> This PR fixes issue #479 where the 'Drag body' label appears on a rigid body if the Variant variables are not yet defined. This causes confusion and can make Simulink bodies appear to be a different type than they actually are.
> 
> This fix uses the 'Allow zero active variant controls' option in the rigid body which allows no label to be assigned until a body is one of the Variants is correctly activated. This requires the library to be in R2017b or later.

@akeeste, can you add a screenshot of what was changed in the library for reference?

kmruehl reviewed (approved):
Thanks @akeeste this PR resolves #479! And thanks for the other cleanup.

akeeste commented:
I'm adding a few screenshots of what has been changed in Simulink for easy reference. The main change that fixed the actual subsystem labeling issue is changing the block parameter option "Allow zero active variant controls". I changed this in the properties table and in the mask:
![image](https://user-images.githubusercontent.com/72414466/102536670-8d3c0300-406f-11eb-82a3-c3213bcdf74e.png)
![image](https://user-images.githubusercontent.com/72414466/102536771-b197df80-406f-11eb-894f-d31c116957dd.png)


As a small clean-up item, I also changed the annotations for the excitation force in each case to reflect that the mean drift force is also included. For example:
![image](https://user-images.githubusercontent.com/72414466/102536992-fde31f80-406f-11eb-9957-51d060a46b34.png)

This is done for each of the F_exc types:
![image](https://user-images.githubusercontent.com/72414466/102537111-266b1980-4070-11eb-9da7-5c177a9b6ecd.png)
